docs/_includes/ds6/button-cta.html

```diff
@@ -103,13 +103,13 @@ <h4>Purple Background (tbd)</h4>
 
 <h4>Button overflow behavior</h4>
 
-<p>Text of buttons with a fixed width (or max-width) will be truncated by default.</p>
+<p>Text of buttons will overflow like their plain HTML companions.</p>
 
 <div class="demo">
     <div class="demo__inner" style="max-width: 200px">
```

Comment:
Do we want to add a class for different widths that people can use? Or are we leaving it up to the people using this to specify the given max width. 
It just seems off to specify it inline, but maybe it's only for this example.

Reply:
This is just for this example in order to force it to wrap. Generally, some outer element would be enforcing a width on the button, causing it to wrap. Hence, the inline style on the demo element.